frontend/src/views/account-mgmt/ManageUsersView.vue

```diff
@@ -276,6 +276,44 @@ export default {
     isExpenseAuthority(user) {
       return user.facilities.some((facility) => facility.isExpenseAuthority) ? 'Yes' : 'No'
     },
+
+    /**
+     * Deactivates a user if they are not the last Expense Authority for any facility.
+     */
+    deactivateUser(user) {
+      const { isLastExpenseAuthority, facilityNames } = this.isUserLastExpenseAuthority(user.contactId);
+      if (isLastExpenseAuthority) {
+        this.setFailureAlert(`Cannot deactivate user. They are the last expense authority for facilities: ${facilityNames.join(', ')}.`);
+      } else {
+        this.toggleDeactivateUserDialog(user);
+      }
+    },
+
+    /**
+     * Checks if the user is the last Expense Authority for facilities they are currently assigned.
+     * Returns an object with a boolean indicating if they are the last expense authority, and an array of facility names.
+     */
+    isUserLastExpenseAuthority(contactId) {
+      // Find target users's facilities that are marked as isExpenseAuthority
+      const targetFacilities = this.usersAndFacilities.find(user => user.contactId === contactId)?.facilities.filter(f => f.isExpenseAuthority) || [];
+      // If no facilities with isExpenseAuthority true, return false
+      if (targetFacilities.length === 0) {
+        return { isLastExpenseAuthority: false, facilityNames: [] };
+      }
+      const lastExpenseAuthorityFacilityNames = [];
+      // Check for each of the target's facilities with isExpenseAuthority true
+      for (const facility of targetFacilities) {
+        // Check if any other user has a facility with the same facilityId and isExpenseAuthority true
+        const hasOtherUserWithAuthority = this.usersAndFacilities.some(user =>
+          user.contactId !== contactId &&
+          user.facilities.some(f => f.facilityId === facility.facilityId && f.isExpenseAuthority));
+        if (!hasOtherUserWithAuthority) {
+          lastExpenseAuthorityFacilityNames.push(facility.facilityName);
+        }
+      }
+      const isLastExpenseAuthority = lastExpenseAuthorityFacilityNames.length > 0;
+      return { isLastExpenseAuthority, facilityNames: lastExpenseAuthorityFacilityNames };
```

Comment:
I'm not sure there's any benefit to returning an object here when one the values is just derived from the other. Your calling code can just test the array length.